src/peft/tuners/lora/__init__.py

```diff
@@ -14,7 +14,7 @@
 
 from peft.import_utils import is_bnb_4bit_available, is_bnb_available, is_eetq_available
 
-from .config import LoftQConfig, LoraConfig, LoraRuntimeConfig
+from .config import LoftQConfig, LoraConfig, LoraPlusConfig, LoraRuntimeConfig
```

Comment:
Hmm, do we really need it here? I think it's enough to have this in the global PEFT namespace.